index.html

```diff
@@ -47,16 +47,27 @@
       <div class="container">
         <h1 class="welcome-title">Welcome to Real Dev Squad</h1>
 
-        <img
-          class="logo-img"
-          src="./img/logos/Real-Dev-Squad@1x.png"
-          alt="Real Dev Squad logo"
-        />
-        <h3 class="info-note">
-          To know how this page is getting developed, join our Discord Channel. Contact
-          one of our existing <a href="https://members.realdevsquad.com/">members</a> for
-          the invitation link.
-        </h3>
+        <div class="cards-title">
+          <h3>Check out our Docs</h3>
+        </div>
+        <div class="cards">
+          <div class="card-1 card">
```

Comment:
Are we defining the class as card-1 card-2 etc ., Anywhere?

Reply:
> Are we defining the class as card-1 card-2 etc ., Anywhere?

No I added these classes for future purpose like if needed to add different data in the cards. Should I remove them?